interface/urls.py

```diff
@@ -10,6 +10,7 @@
     url(r'^repo/(?P<full_name>[\w/.-]+)/delete$', views.RepoDeleteView.as_view(), name='repo_delete'),
     url(r'^repo/(?P<full_name>[\w\-_]+/[\w\-_]+)/(?P<path>[\w/\-._]+)?$',
         views.RepoDetailView.as_view(), name='repo_detail'),
+    url(r'^search/(?P<full_name>[\w\-_]+/[\w\-_]+)', views.search_view, name='search'),
```

Comment:
Why do we care about the repo name here, while we are searching over all the documents?